src/pages/Dashboard/index.tsx

```diff
@@ -1,19 +1,72 @@
 /** @jsxImportSource @emotion/react */
-import React from 'react';
+import React, { useMemo } from 'react';
+
+import { useIsLgDown, useIsSmDown } from 'hooks/responsive';
+import { Tabs } from 'components';
+import { useTranslation } from 'translation';
 import MyAccount from './MyAccount';
+import MintRepayVai from './MintRepayVai';
 import { SupplyMarket, BorrowMarket } from './Markets';
 import { useStyles } from './styles';
 
 const DashboardUi: React.FC = () => {
+  const { t } = useTranslation();
+  const isLgDown = useIsLgDown();
+  const isSmDown = useIsSmDown();
   const styles = useStyles();
+
+  const tabsContent = useMemo(
+    () => [
+      {
+        title: t('dashboard.markets.tabSupply'),
+        content: <SupplyMarket css={styles.item} />,
+      },
+      {
+        title: t('dashboard.markets.tabBorrow'),
+        content: <BorrowMarket css={styles.item} />,
+      },
+    ],
+    [],
+  );
+
+  const renderMarkets = () => {
```

Comment:
Functions labeled as render can usually be better written as a component with props

Reply:
refactored